homeassistant/components/zoneminder.py

```diff
@@ -40,6 +41,27 @@
 }, extra=vol.ALLOW_EXTRA)
 
 
+class _Feature:
+    def __init__(self, version_str):
+        self._version = packaging.version.parse(version_str)
+
+    def is_supported(self):
+        """Returns False if this feature isn't supported on the currently
+connected zoneminder instance. Otherwise returns True.
+
+        """
+        remote_version = ZM.get('version', None)
+        if remote_version is None:
+            # Be optimistic, assume the remote end supports the feature.
+            return True
+        return self._version <= remote_version
+
+# Introduced in 1.30:
+#
+# https://github.com/ZoneMinder/ZoneMinder/commit/2888142e682bbc9950535d7e5aaef2cd20cda38d
+ALARM_STATUS_FEATURE = _Feature("1.30")
```

Comment:
Fixed. I now just need to remember that flake8 is used as one of the linters and not pyflakes...